cmd/evm/internal/t8ntool/execution.go

```diff
@@ -225,7 +225,7 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 	// Commit block
 	root, err := statedb.Commit(chainConfig.IsEIP158(vmContext.BlockNumber))
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Could not commit state: %v", err)
+		fmt.Fprintf(os.Stderr, "could not commit state: %v", err)
```

Comment:
So, usually, we make `error`s lowercase, because they are usually wrapped within some other error or context before shown to the user. 
This, however, is a message printed on stderr, so will not be further wrapped, and should therefore be a proper sentence. So please revert this

Reply:
Fixed. Thanks for the clarification!